client/src/index.jsx

```diff
@@ -15,7 +13,8 @@ class App extends React.Component {
             index: 0,
             imageData: [],
             image: [],
-            popUp: false
+            popUp: false,
+            dataLoaded: false
```

Comment:
This following comment has nothing to do with dataLoaded state: Why is your App component inside the index.jsx file? Do you want to make an App.jsx file and just import App into this file? By convention, we name .jsx files according to the component name.